ios/RNRadarTests/RNRadarUtilsTests.m

```diff
@@ -73,12 +73,13 @@ - (void)testPlacesProviderForString {
 }
 
 - (void)testOptionsForDictionary {
-    NSDictionary *optionsDict = @{@"sync": @"all", @"offline": @"replayOff", @"invalid": @"ingore me"};
+    NSDictionary *optionsDict = @{@"sync": @"all", @"offline": @"replayOff", @"invalid": @"ingore me", @"priority":@"responsiveness"};
```

Comment:
@brettguenther Can you add a space `@"priority": @"responsiveness"`? That will also trigger another build which should cause checks to pass now.